modules/simple_forms_api/app/controllers/simple_forms_api/v1/uploads_controller.rb

```diff
@@ -22,8 +23,41 @@ class UploadsController < ApplicationController
       def submit
         Datadog::Tracing.active_trace&.set_tag('form_id', params[:form_number])
 
-        form_id = FORM_NUMBER_MAP[params[:form_number]]
+        if form_is210966 && icn
+          handle_210966_authenticated
+        else
+          submit_form_to_central_mail
+        end
+      rescue => e
+        raise Exceptions::ScrubbedUploadsSubmitError.new(params), e
+      end
+
+      def authenticate
```

Comment:
`authenticate` will normally raise an exception so we want to bypass that. We're really just using `authenticate` to populate `@current_user`.